compositor_pipeline/src/pipeline/decoder/ffmpeg_h264.rs

```diff
@@ -133,8 +135,8 @@ fn chunk_to_av(chunk: EncodedChunk) -> Result<ffmpeg_next::Packet, DecoderChunkC
     let mut packet = ffmpeg_next::Packet::new(chunk.data.len());
 
     packet.data_mut().unwrap().copy_from_slice(&chunk.data);
-    packet.set_pts(Some(chunk.pts));
-    packet.set_dts(chunk.dts);
+    packet.set_pts(Some(chunk.pts.as_millis() as i64));
```

Comment:
add some comment describing that using millisecond here and setting time base to 1:1000 are connected.

Also, original pts are defined with a precision of about 0.01 ms and you are lowering that precision to 1 ms. I don't think it will cause anything noticable, but if it's just as easy to pick any other number than why not use  micro seconds ?

Reply:
Or alternative approach export 1_000_000 as a const, and do use from_secs_f64/as_secs_f64 and multiply/divide yourself. With this approach, some comments would still be useful in this approach too.

Reply:
the same applies to other decoders and encoders